### PR TITLE
Feature/database connection

### PR DESCRIPTION
**Pull request by SlimPortfolio**
set up database connections

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,23 +1,31 @@
 const express = require('express');
-const bodyParser = require('body-parser');
+// const bodyParser = require('body-parser');
```
Comment (StreetsJ) [inline server/server.js:2]: This can be removed

File: server/server.js [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+var mysql = require('mysql');
 const { Client } = require('pg');
 const app = express();
 const port = 3001; //TODO
 
 // Add the bodyParser middelware to the express application
-app.use(bodyParser.urlencoded({ extended: false }));
+// app.use(bodyParser.urlencoded({ extended: false }));
```
Comment (StreetsJ) [inline server/server.js:9]: This can be removed

File: server/server.js [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
-  database: process.env.DB_DATABASE
-})
+// const client = new Client({
+//   host: process.env.DB_HOST,
+//   user: process.env.DB_USER,
+//   port: process.env.DB_PORT,
+//   password: process.env.DB_PASS,
+//   database: process.env.DB_DATABASE
+// })
+
```
Comment (StreetsJ) [inline server/server.js:22]: These can be removed

File: server/server.js [hunk 4]
```diff
@@ -1,23 +1,31 @@
...
+var client = mysql.createConnection({
+  user: 'root',
+  password: "password",
+  socketPath: "MySQL",
+  database: "event_planner"
```
Comment (StreetsJ) [inline server/server.js:27]: This has been moved to the .env right?

Comment (SlimPortfolio) [reply server/server.js:27]: these have been resolved. The whole server.js file was deleted and replaced with database.js

**Pull request merged**